src/style.css

```diff
@@ -1,3 +1,54 @@
 body {
   margin: 0;
```

Comment:
Avoid using tag names for styling (except `html` and `body`). Style all elements using `.class` and if needed with `:pseudo-class`, `pseudo-element` and `[attribute]`.